### ISSUE TITLE
faceapp.py not exists in the project

### ISSUE THREAD

**Issue by yossimiz** [Alloooshe/facelib_modular_face_recognition_pipline#1]
the file faceapp.py not exists in the project or instructions of how to use and run the GUI 
it also mentioned it can use different platform, no info :-(

**Comment by Alloooshe** [comment]
yes, I didn't upload the UI, the main objective of this repo is to provide general framework and not ready to use product, you can still call the functions from facecore.py and use that in your application

**Comment by yossimiz** [comment]
But also you didn't add instructions. I had error running and I fixed them
all but I don't know how to activate face recognition. I'm calling to
facecore.py and nothing happen.
So example will help. Meanwhile I move on to other examples....

Anyway, if you are interesting I will pay for you to adapt the code to my
needs. Let me know if you are interesting?


בתאריך יום ב׳, 15 במרץ 2021, 17:56, מאת Ali Mohammad ‏<
***@***.***>:

**Comment by Alloooshe** [comment]
you can contact me on my email. ali.mhd.work@gmail.com

**Comment by Alloooshe** [comment]
I will try to update the functionality and seperate the pakcage from UI and add clearer instructions
